### ISSUE TITLE
Autofill accessibility is broken on the latest Android 12

### ISSUE THREAD

**Issue by Ulrar** [bitwarden/mobile#1432]
## Describe the Bug

The Autofill feature (using the tile or popup through accessibility) doesn't work since the latest Android 12 update last week.
I was already on the previous Android 12 beta beforehand and had no issues so they must have changed something in the latest build.

## Steps To Reproduce

1 - Enroll in the Android 12 beta.
2 - Install Bitwarden
3 - Try to enable "Use accessibility" under Autofill

## Expected Result

Enabling autofill should work

## Actual Result

Enabling autofill does not work, Android says Bitwarden is malfunctioning.

## Screenshots or Videos

Taking screenshots of the Bitwarden app isn't allowed, apparently. But the first two auto fill buttons do work fine (autofill service and inline autofill), but the third one (use accessibility) that makes the tile work brings you to an Android settings menu saying Bitwarden is malfunctioning.

## Environment

- Device: Pixel 3 XL
- Operating system: Android 12, latest update
- Build Version : 2.10.0 (3516)
- Is this a Beta release? : Of android yes

## Additional Context

The keyboard suggestions do work, but as usual they rarely trigger so I end up having to copy / paste from the app manually most of the time.
The Autofill tile was the only method I found to (mostly) get autofill to work.

**Comment by Ulrar** [comment]
This seems resolved on the latest beta
